test/Application.UTest/Characters/GetTopUserCharacterCompetitiveRatingQueryTest.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Crpg.Application.Characters.Queries;
+using Crpg.Application.Common.Results;
+using Crpg.Domain.Entities.Characters;
+using NUnit.Framework;
+
+namespace Crpg.Application.UTest.Characters;
+
+public class GetTopUserCharacterCompetitiveRatingQueryTest : TestBase
+{
+    [Test]
+    public async Task ShouldReturnErrorIfCharacterRatingDoesntExist()
```

Comment:
Not sure what you mean here. I don't think that query should ever return an error.